godtools/App/Services/Localization/LocalizationServices.swift

```diff
@@ -109,4 +109,9 @@ class LocalizationServices {
         
         return key
     }
+    
+    func toolCategoryStringForBundle(bundle: Bundle, attrCategory: String) -> String {
```

Comment:
Hey @rachaelblue at first I wasn't sure about this change with it being viewModel (domain layer) logic, but then I realized most services have a mix of both domain layer and data layer logic already.
I'm going to think about that more and the possibility of creating more distinction between data layer logic and domain layer logic.  It would involve us either creating more use cases or viewModel extensions for sharing domain logic between viewModels.

Reply:
Yeah, I wasn't sure either 😄  The issue I saw was that we have a fair amount of logic that is repeated between viewModels.  You're right, it _is_ domain layer logic, so it is correct that it's in the viewModel, but it seems like there should be a separate place to store domain layer logic that multiple viewModels use.  Some of the logic is probably so ubiquitous that putting it in an extension or useCase seems like overkill to me-- maybe just have some static methods stashed somewhere central?

Reply:
For now I think the best thing is to create that Share/Domain folder and organize domain logic in there, that way it's easy to locate since there isn't a standard way to share domain logic.  It's a bit mixed at the moment.  So it will either live there or in Services at the moment. https://jira.cru.org/browse/GT-1624